sgl-kernel/csrc/cpu/vec.h

```diff
@@ -6,6 +6,7 @@
 
 #include <ATen/cpu/vec/functional.h>
 #include <ATen/cpu/vec/vec.h>
+#include <ATen/native/cpu/utils.h>
```

Comment:
```suggestion

```

Reply:
ok, then put it in `fla.cpp`, i tried to isolate `cpu/utils.h` from torch with sgl-kernels.